packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewSummary.tsx

```diff
@@ -118,6 +120,17 @@ export const TransactionReviewSummary = ({
                 </Note>
             )}
 
+            {connectPopupCall?.state === 'ongoing' && (
+                <Note iconName="plug">
+                    <Translation id="TR_CONNECTED_TO" />
+                    {': '}
+                    <Text variant="primary">
+                        {connectPopupCall.source?.manifest?.appName ||
```

Comment:
I would like to make manfiest.appName is already required, for connect https://github.com/trezor/trezor-suite/blob/feat/suite-connect-sign-modals/suite-common/connect-popup/src/connectPopupTypes.ts#L6

are you planning to make it required also for walletconnect?

Reply:
Yes, I will add it for WalletConnect too